products/bluebell/core/src/evm_bytecode_generator.rs

```diff
@@ -463,7 +457,7 @@ impl<'ctx> EvmBytecodeGenerator<'ctx> {
                                     evm_block.push_u256(address);
                                     evm_block.set_next_rust_position(file!().to_string(), line!() as usize);
                                     evm_block.external_sload();
-                                    evm_block.register_stack_name(value_name);
+                                    let _ = evm_block.register_stack_name(value_name);
```

Comment:
Why does this need an assignment?

Reply:
This is to silence the compilers "unused warning" (recommended modification by the compiler).